storage.go

```diff
@@ -42,10 +42,24 @@ func (s *Storage) createDir(ctx context.Context, path string, opt pairStorageCre
 
 	rp := s.getAbsPath(path)
 
-	// Add `/` at the end of path to simulate directory.
+	// Add `/` at the end of `path` to simulate a directory.
 	// ref: https://developer.qiniu.com/kodo/kb/1705/how-to-create-the-folder-under-the-space
 	rp += "/"
 
+	// kodo `put` doesn't support `overwrite`, so we need to check and delete the dir if exists.
```

Comment:
Maybe we should omit resource  exist error here and there's not need to delete and recreate the dir?

Reply:
This seems reasonable (but not specified?). Sounds like `mkdir -p` (existing directory and contents are preserved)